ksqldb-execution/src/main/java/io/confluent/ksql/execution/ddl/commands/CreateSourceCommand.java

```diff
@@ -94,6 +95,9 @@ public Boolean getIsSource() {
   }
 
   private static void validate(final LogicalSchema schema, final boolean windowed) {
+    if (schema.valueContainsAny(SystemColumns.windowBoundsColumnNames())) {
```

Comment:
What is the purpose of this change (from `systemColumnNames()` to `windowBoundsColumnNames()`)?

Reply:
This is one of the removed redundant validation steps we talked about, but the validation for window columns was necessary IIRC (also doesn't require piping a config, so no extra work).

Reply:
Why is the validation for window bounds column names different from that for other system column names? Are there other places where we check for all system column names except window bounds column names? That feels very strange.

Reply:
Now that I worked with the code more, I vaguely remember leaving this in just because I felt like the extra check was good. Even if that's a false memory, I tested and this check is indeed unnecessary. Now that I think about it more, I think it's better we fail columns aliased with window bound column names in the same place as we do other system columns (which I'm guessing is your conclusion too?).

Reply:
If our existing test coverage for this (aliasing columns with a window bound name) is good and the removal of this check doesn't break tests, then yes I think it's preferable to keep all the system column checks in the same place. (To clarify, I don't know the state of existing test coverage for this scenario -- it'd be good to check and add tests for it if it's lacking.)

Reply:
We have coverage in `system-columns.json` and `CreateSourceFactoryTest`.